ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -128,11 +129,13 @@ struct MSFListCellView: View {
 
 struct ListCellButtonStyle: ButtonStyle {
     let tokens: MSFListCellTokens
-    let layoutType: MSFListCellLayoutType
+    let state: MSFListCellState
```

Comment:
Have you tested changing the state.subtitle property at runtime and see if the cell updates?
I'll be good to test both the cell height and the display of the subtitle itself.

Reply:
The cell updates as expected. If the subtitle exists but the title is empty, it should retain a 2-line state. If the subtitle is removed, the cell should go back to a 1-line state :)